.github/workflows/ci.yml

```
@@ -0,0 +1,83 @@
+name: CI
+on:
+  - push
+  - pull_request
+
+jobs:
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    name: ${{ matrix.os }} - Atom ${{ matrix.atom_channel }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+         # - macos-latest
+         # - windows-latest
+        atom_channel: [stable, beta]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: UziTech/action-setup-atom@v1
+        with:
+          channel: ${{ matrix.atom_channel }}
+      - name: Versions
+        run: |
+          apm -v
+      - name: Install APM dependencies
+        run: |
+          apm ci  # uses locked module. use `apm install` for non-locked
+      - name: Atom Package dependencies
+        run: |
+          apm install atom-ide-markdown-service
+          apm install busy-signal
+      - name: Run tests 👩🏾‍💻
+        run: npm run test
+
+  Lint:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Commit lint ✨
+        uses: wagoid/commitlint-github-action@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "13.x"
+      - name: Install NPM dependencies
+        run: |
+          npm ci  # uses locked module. use `npm install` for non-locked
+      - name: Lint ✨
+        run: npm run lint
+
+  Release:
+    needs: [Test, Lint]
+    if: github.ref == 'refs/heads/master' &&
+      github.event.repository.fork == false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: UziTech/action-setup-atom@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: NPM install
+        run: npm ci
+      - name: Release 🎉
+        uses: cycjimmy/semantic-release-action@v2
+        with:
+          extends: |
+            @semantic-release/apm-config
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.ATOM_ACCESS_TOKEN  }}
```

Both need to use `ATOM_ACCESS_TOKEN`, otherwise it won't be picked up by `@semantic-release/apm-config`.

```suggestion
          ATOM_ACCESS_TOKEN: ${{ secrets.ATOM_ACCESS_TOKEN  }}
```

---

Fixed in https://github.com/atom-ide-community/atom-ide-datatip/pull/66/commits/c611ada7745d4635df9630fb349b0d6932211083